devito/ops/transformer.py

```diff
@@ -1,4 +1,4 @@
-def make_ops_ast(expr, nfops):
+def make_ops_ast(expr, nfops, is_write=False):
     """
     Transform a devito expression into an OPS expression.
```

Comment:
Devito, Expression?

Reply:
I think 'expression' is ok because it's loosely referring to a fairly generic expression